source/parameters/bend.md

```diff
@@ -0,0 +1,193 @@
+hence will move all downstream lattice elements in space.
+
+  The total bend strength felt by a particle is the vector sum of `g` plus the zeroth order
+magnetic multipole. If the multipole `tilt0` and `Ks0` is zero, the total bend strength is
+  ```{code} yaml
+  norm_bend_field = g + Kn0
+  ```
+  Changing the multipole strength `Kn0` or `Ks0` leaves the Reference orbit and the positions of
+all downstream lattice elements
+unchanged but will vary a particle's orbit. One common mistake when designing lattices is to vary
+`g` and not `Kn0` which results in downstream elements moving around. See Sref{s:ex.chicane}
+for an example.
+
+  Note: A positive `g`, which will bend particles and the reference orbit in the {math}`-x` direction
+represents a field of opposite sign as the field due a positive `hkick`.
+%
+- **h1, h2**
+The attributes `h1` and `h2` are the curvature of the entrance and exit pole faces.
+%
+- **L, L_arc, L_chord, L_sagitta (output param)** 
+The `Length` parameter, is the arc length of the reference trajectory through the bend.
+
+  `L_chord` is the chord length from entrance point to exit point.
+The `L_sagitta` parameter is the sagitta length (The sagitta is the distance
+from the midpoint of the arc to the midpoint of the chord). `L_sagitta` can be negative and will have
+the same sign as the `g` parameter.
+%
+- **L_rectangle**
+The `L_rectangle` parameter is the "rectangular" length defined to be the distance between the
+entrance and exit points. The coordinate system used for the calculation is defined by the setting
+of `fiducial_pt`. {numref}`f:rbend` shows `l_rectangle` for `fiducial_pt` set to
+`entrance_end` (the coordinate system corresponds to the entrance coordinate system of the bend).
+In this case, and in the case where `fiducial_pt` is set to `exit_end`, the rectangular
+length will be {math}`rho sinalpha`. If `fiducial_pt` is set to `none` or `center`,
+`l_rectangle` is the same as the chord length.
+%
+- **ref_tilt**
+The `ref_tilt` attribute rotates a bend about the longitudinal axis at the entrance face of the
+bend. A bend with `ref_tilt` of {math}`\pi/2` and positive `g` bends the element in the {math}`-y`
+direction ("downward"). See {numref}`f:tilt.bend`. It is important to understand that `ref_tilt`,
+unlike the `tilt` attribute of other elements, bends both the reference orbit along with the
+physical element. Note that the MAD `tilt` attribute for bends is equivalent to the Bmad
+`ref_tilt`. Bends in Bmad do not have a `tilt` attribute.
+
+  Important! Do not use `ref_tilt` when doing misalignment studies for a machine. Trying to misalign
+a dipole by setting `ref_tilt` will affect the positions of all downstream elements! Rather, use the
+`tilt` parameter.
+
+%---------------
+
+  The attributes `g`, `angle`, and `L` are mutually dependent. If any two are specified for
+an element AcceleratorLattice will calculate the appropriate value for the third.
```

Comment:
What is the object `AcceleratorLattice` that is referenced here?  Is this defined elsewhere, or does this just refer to some logic in the Translator layer?

Reply:
Typo. Fixed.